config/standard/contact-summary.templated.js

```diff
@@ -11,6 +11,27 @@ var isFacilityDelivery = extras.isFacilityDelivery;
 var getBirthDate = extras.getBirthDate;
 var isHighRiskPostnatal = extras.isHighRiskPostnatal;
 var addImmunizations = extras.addImmunizations;
+var getMostRecentNutritionEnrollment = extras.getMostRecentNutritionEnrollment;
+var countNutritionFollowups = extras.countNutritionFollowups;
+var isActivePregnancy = extras.isActivePregnancy;
+var getSubsequentVisits = extras.getSubsequentVisits;
+var isHighRiskPregnancy = extras.isHighRiskPregnancy;
+var getSubsequentDeliveries = extras.getSubsequentDeliveries;
+var getOldestReport = extras.getOldestReport;
+var getSubsequentPregnancies = extras.getSubsequentPregnancies;
+var getAgeInMonths = extras.getAgeInMonths;
+var initImmunizations = extras.initImmunizations;
+var isVaccineInLineage = extras.isVaccineInLineage;
+var isSingleDose = extras.isSingleDose;
+var countDosesReceived = extras.countDosesReceived;
+var countDosesPossible = extras.countDosesPossible;
+var countReportsSubmittedInWindow = extras.countReportsSubmittedInWindow;
+var getMostRecentReport = extras.getMostRecentReport;
+var getTreatmentEnrollmentDate = extras.getTreatmentEnrollmentDate;
+var pregnancyForms = extras.pregnancyForms;
```

Comment:
Sorry about all this boiler-plate. This will be fixed when we support ES6 in the v3.7 release.